operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/MetadataAwareOnUpdateFilter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates and other contributors as indicated by
```

Comment:
I guess this copyright part can be removed

Reply:
Yes, sorry just copied this from what was in keycloak.